### ISSUE TITLE
Allow selecting deleted text in inline view

### ISSUE THREAD

**Issue by dsherret** [microsoft/vscode-pull-request-github#6001]
- Extension version: v0.88.1
- VSCode Version: 1.89.1
- OS: Windows 11

Steps to Reproduce:

1. Switch diff to inline view
2. Go to a block of text that's been deleted and try to select a word

Instead of being able to select specific text, a context menu appears:

![image](https://github.com/microsoft/vscode-pull-request-github/assets/1609021/1de7e929-3326-4176-a15d-131e4d7c559e)

Video:

https://github.com/microsoft/vscode-pull-request-github/assets/1609021/0db6baa9-1bde-43ea-9296-c428b9c6b446

**Comment by alexr00** [comment]
Yep, this is the behavior of VS Code. https://github.com/microsoft/vscode/issues/8226
